src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/Designer/DataTransferTypes.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug.Designer
+{
+    internal sealed record PropertyValueChangeRequest(
+        string ProfileName,
+        string PropertyName,
+        object Value);
+
+    internal sealed record Profile(string Name, string DisplayName);
```

Comment:
Should the `Profile` include the debug command name? In the current design it is not an editable property, but it would be useful to display to the user.

Reply:
The command name is not localizable, but we could associate it with a localizable string.

Reply:
OK, the `ProfileType` now associates the command name with a localizable string.